docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -13,106 +13,121 @@ aliases:
 
-## Controlling your mock robot using a Viam SDK
+## Control your mock robot using a Viam SDK
 
-### How to install a Viam SDK
+### Install a Viam SDK
 
-In this step, you are going to install either the [Viam Python SDK](https://python.viam.dev/) (Software Development Kit) or the [Viam Go SDK](https://pkg.go.dev/go.viam.com/rdk). Use which ever programming language you are most comfortable with.
+In this step, you will install either the [Viam Python SDK](https://python.viam.dev/) or the [Viam Go SDK](https://pkg.go.dev/go.viam.com/rdk).
+Use which ever programming language you are most comfortable with.
 
-{{% alert title="Note" color="note" %}}
-
-Refer to the appropriate SDK documentation for SDK installation instructions.
+Refer to the appropriate SDK documentation for SDK installation instructions:
 
 - [Viam Python SDK](https://python.viam.dev/)
-- [Viam Go SDK](https://pkg.go.dev/go.viam.com/rdk)
-
-{{% /alert %}}
+- [Viam Go SDK](https://github.com/viamrobotics/rdk/tree/main/robot/client)
 
-{{% alert title="Tip" color="tip" %}}
-{{< snippet "social.md" >}}
-There, you will find a friendly developer community of people learning how to make robots using Viam.
-{{% /alert %}}
+### Connect to your mock robot with your code
 
-### How to connect to your mock robot with the Viam SDK
+The easiest way to get started writing an application with Viam's SDKs is to use the boilerplate code on the **Code Sample** tab.
 
-The easiest way to get started writing an application with Viam is to navigate to the [robot page on the Viam app](https://app.viam.com/robots), select the **code sample** tab, and copy the boilerplate code from the section labeled **Python SDK** or **Go SDK**.
-These code snippets import all the necessary libraries and set up a connection with the Viam app in the cloud.
-Next, paste that boilerplate code from the **code sample** tab of the Viam app into a file named <file>index.py</file> or <file>index.go </file>file in your code editor, and save your file.
+Navigate to your [robot's page on the Viam app](https://app.viam.com/robots), select the **Code Sample** tab, and select your SDK language (**Python** or **Golang**).
+Copy the boilerplate code.
+This code snippets imports all the necessary libraries, is pre-populated with your robot credentials, and sets up a connection with the Viam app in the cloud.
+Next, paste that boilerplate code into a file named <file>index.py</file> or <file>index.go</file> in your code editor, and save your file locally.
 
 {{< readfile "/static/include/snippet/secret-share.md" >}}
 
 You can now run the code.
 Doing so verifies that the Viam SDK is properly installed, that the `viam-server` instance on your robot is live, and that the computer running the program is able to connect to that instance.
 
-You can run your code by typing the following into the terminal:
+Run your code by typing the following into the terminal on your computer:
```

Comment:
you'd need to still hit enter
```suggestion
Run your code by entering the following into the terminal on your computer:
```

Reply:
good point

Reply:
I mean the Setup tab tells them to but I should probably clarify within the tutorial itself.